src/js/base/Context.js

```diff
@@ -195,7 +195,7 @@ export default class Context {
   }
 
   /**
-   * Some buttons need to change their visual style immediately once they get pressed
+   *Some buttons need to change their visual style immediately once they get pressed
```

Comment:
Please roll back the space.

Reply:
I can, I wasn't aware that had happened, lol.